config/default.json

```diff
@@ -1,10 +1,10 @@
 {
   "endpoints": {
-    "api":"https://automation-staging.atomist.services/registration",
-    "graphql": "https://automation-staging.atomist.services/graphql/team"
+    "api": "https://automation.atomist.com/registration",
+    "graphql": "https://automation.atomist.com/graphql/team"
```

Comment:
These changes will break the staging.conf. Did you mean to commit these? I typically create a new conf and use that to test in a specific team.